### ISSUE TITLE
Save to gallery feature

### ISSUE THREAD

**Issue by s0mt0chukwu** [tom-anders/Easy_xkcd#338]
A Save to option or download button should be included in the next update

**Comment by tom-anders** [comment]
Thanks for the suggestion, but shouldn't that already be possible via the "share" button?

**Comment by tom-anders** [comment]
Right, but for example on my device (Pixel) I can share to Google Photos and then it will appear in my gallery.
